tests/bettertransformer/testing_bettertransformer_utils.py

```diff
@@ -70,7 +70,7 @@ def test_logits(self):
                 torch.manual_seed(0)
                 bt_hidden_states = converted_model(**inputs)[0]
 
-                if "gelu_new" in vars(random_config).values():
+                if "gelu_new" in list(random_config.to_dict().values()):
```

Comment:
Great change, but no need to cast to `list` here!

Reply:
thanks! but I think the `to_dict()` is necessary (otherwise I get a VERY weird error from `transformers`)

Reply:
Yes exactly, but accessing the value is enough here